sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRestClient.cs

```diff
@@ -12,157 +12,157 @@ namespace Azure.AI.TextAnalytics
 {
     internal partial class TextAnalyticsRestClient
     {
-        private HttpMessage CreateHealthStatusNextPageRequest(string apiversion, string nextLink, bool? showStats)
```

Comment:
this is the method I manually did because the swagger didn't have the `x-ms-pagination` on it